documentation/wiki/UnGAC.md

```diff
@@ -0,0 +1,28 @@
+# MSBuild, the Global Assembly Cache (GAC), and You
+
+## What is the GAC?
+
+See the [public documentation](https://docs.microsoft.com/dotnet/framework/app-domains/gac). The GAC is a folder where different installations of VS on the same machine look for assemblies that are commonly used. If an assembly is in the GAC, it will be prioritized over any other assembly.
+
+The only MSBuild assemblies you may see in the GAC are version 4.0. There is no reason any modern MSBuild assembly should be in the GAC today.
```

Comment:
Isn't it version 4.8?

Reply:
And maybe define "modern" (15.0+)

Reply:
You're right, folders are marked as 4.0 but the assemblies themselves are 4.8.4